### ISSUE TITLE
Feature Request: List Proxy

### ISSUE THREAD

**Issue by ricochet1k** [CyclopsMC/IntegratedDynamics#192]
While trying to figure out how I could build a custom auto-crafting system with Integrated Dynamics, I realized that having a List Proxy that could store a list of recipe variables would be really handy. I'd love to be able to add/remove variables from the list without having to rewrite a variable.

Also, I haven't yet checked if proxy/materializers can be used by pipes/item tunnels, but that would be great too.

**Comment by rubensworks** [comment]
Could you clarify what you mean with a short example? Because I'm not entirely sure what you mean.
As recipe variables don't exist atm, or are you requesting that as well?

**Comment by ricochet1k** [comment]
Nah, you don't need recipe variables, List of Items works just fine.

**Comment by ricochet1k** [comment]
So, it would work like a proxy, in that you can create variables that refer to the List Proxy Block and indirectly to the variables inside it. You could then add/remove variables from the inventory, and the values of those variables would appear in the list.

**Comment by rubensworks** [comment]
Ah ok, now I understand.
Good idea!
Shouldn't be too hard to implement.
Can't say when I'll be able to do it though.
